examples/firebase-functions/README.md

```diff
@@ -0,0 +1,37 @@
+# Firebase function example
+
+#### Install dependencies / Setup
```

Comment:
Readme must be the same style as any other example in the project

and instructions must use `pnpm` instead of `npm`